lib/auth/windows/windows.go

```
@@ -77,6 +77,21 @@ func getCertRequest(username, domain string, clusterName string, ldapConfig LDAP
 			san,
 		},
 	}
+
+	if req.ActiveDirectorySID != nil {
+		adUserMapping, _ := asn1.Marshal(SubjectAltName[adSid]{
```

Are we intentionally ignoring an error here?

---

No, fixed to return error